src/resources/js/aggregator-fields.js

```diff
@@ -100,7 +101,16 @@ tribe_aggregator.fields = {
 			.on( 'click'      , obj.selector.view_filters              , obj.events.toggle_view_filters )
 			.on( 'blur'       , obj.selector.datepicker                , obj.date_helper )
 			.on( 'submit'     , obj.selector.tab_new                   , obj.events.suppress_submission )
-			.on( 'change'     , obj.selector.import_type_field         , obj.reset_preview )
+			.on( 'change'     , obj.selector.import_type_field         , function() {
+				// Resets the Preview
+				obj.reset_preview()
+
+				// Every time you change Type of import we reset the frequency field
+				var $this = $( this ),
+					$frequency = $( this ).next( obj.selector.fields );
```

Comment:
Indent using the same number of tabs as the previous line, then spaces to align `$frequency` with `$this`?